src/Form/Form.js

```diff
@@ -72,7 +77,7 @@ class Form {
             window.removeEventListener('mousedown', this.removeTooltip, {capture: true})
         }
         this.removeInputHighlight = (input) => {
-            removeInlineStyles(input, getIconStylesAutofilled(input, this.isLogin))
+            removeInlineStyles(input, getIconStylesAutofilled(input))
```

Comment:
This `isLogin` was no longer used, so I removed it in the definition and all calls.